src/Sulu/Bundle/CustomUrlBundle/Tests/Functional/Controller/CustomUrlControllerTest.php

```diff
@@ -711,8 +711,8 @@ public function testCGet($items)
             $this->assertArrayHasKey('creator', $responseData);
             $this->assertArrayHasKey('changer', $responseData);
 
-            $this->assertEquals($requestTime, new \DateTime($responseData['created']), '', 2);
-            $this->assertEquals($requestTime, new \DateTime($responseData['changed']), '', 2);
+            $this->assertEquals($requestTime, new \DateTime($responseData['created']), '', 5);
+            $this->assertEquals($requestTime, new \DateTime($responseData['changed']), '', 5);
```

Comment:
Is it really fixed that way? Haven't we already used `assertGreaterThanOrEqual` in other places, to make sure the test will always succeed, no matter how long the process might hang before?